### PR TITLE
Use latest Chrome, Ruby and PhantomJS on Travis

### PR DESCRIPTION
**Pull request by titusfortner**
Chrome & Chromedriver versions were both very out of date. Phantom code should either get run or deleted, I opted for deleting, any arguments for running it?

### CODE REVIEW

**Comment by p0deje** [comment]
I've actually been thinking that it would be great to run as many browsers as possible. Specs for Phantom fail which means it probably should be reported as Selenium bugs.

**Comment by titusfortner** [comment]
The phantom tests in Selenium are passing, so if they are failing for Watir, we probably need more Selenium tests. :) I thought we were having issues with intermittent failures in phantom, so they weren't actually providing good information, which is why it was commented out. So, yes, let's add them back in and get them working.

**Comment by titusfortner** [comment]
PhantomJS tests are failing because we are using 1.9; 2.0 does not have a linux binary for easy download like 1.9 did, something about issues in the way it is built. I think there is a way we can build it in Travis and use it, maybe with npm, but I don't have time to figure it out. I'm removing the code because commented out code is bad when we have revision control, and we can put it back if/when we get it working. In the meantime, we should be running Ruby 2.2 and the latest Chrome & Chromedriver binaries.

File: .travis.yml [hunk 1]
```diff
@@ -16,12 +18,3 @@ env:
   - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=chrome
   - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=chrome ALWAYS_LOCATE=false
   - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=chrome PREFER_CSS=1 SELECTOR_STATS=1
-  # - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=phantomjs
-  # - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=phantomjs ALWAYS_LOCATE=false
-  # - RAKE_TASK=spec WATIR_WEBDRIVER_BROWSER=phantomjs PREFER_CSS=1 SELECTOR_STATS=1
-  - RAKE_TASK=yard:doctest
```
Comment (p0deje) [inline .travis.yml]: Why?

Comment (titusfortner) [inline .travis.yml]: Because it is bad practice to comment out code? :)
We can add an issue to get Phantomjs working.

Comment (p0deje) [inline .travis.yml]: You remove doctests :)

Comment (titusfortner) [inline .travis.yml]: Oh, that was a mistake. Delete happy mood I guess. :)

Comment (p0deje) [inline .travis.yml]: I'll fix.

**Comment by titusfortner** [comment]
Oh, I haven't merged it yet, I'll rebase and add it in, though.

**Comment by p0deje** [comment]
I'm currently trying to install PhantomJS 2 on Travis, I'll push the changes here if I succeed. Otherwise, let's merge it later today.

**Comment by titusfortner** [comment]
Ah, sounds good. Thanks.

**Pull request merged**